src/main/kotlin/me/partlysanestudios/partlysaneskies/data/pssdata/PublicDataManager.kt

```diff
@@ -58,18 +58,14 @@ object PublicDataManager {
      * @return the current repo's owner
      */
     fun getRepoOwner(): String {
-        return if (PartlySaneSkies.config == null) {
-            "PartlySaneStudios"
-        } else PartlySaneSkies.config.repoOwner
+        return PartlySaneSkies.config.repoOwner?: "PartlySaneStudios"
```

Comment:
Oh nvm since it works on kotlin, this just means we have to translate more, forget what I said

Reply:
Nah I made it so that it's always not null

Reply:
Nono im talking about the PartlySaneSkies.getConfig() stuff, and here I realized its fine on kotlin